src/lib/swapper/swappers/ThorchainSwapper/utils/getQuote/getQuote.ts

```diff
@@ -0,0 +1,71 @@
+import type { Asset } from '@shapeshiftoss/asset-service'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { adapters } from '@shapeshiftoss/caip'
+import type { Result } from '@sniptt/monads'
+import { Err, Ok } from '@sniptt/monads'
+import { bn } from 'lib/bignumber/bignumber'
+import { toBaseUnit } from 'lib/math'
+import type { SwapErrorRight } from 'lib/swapper/api'
+import { makeSwapErrorRight, SwapErrorType } from 'lib/swapper/api'
+import type {
+  ThorchainSwapperDeps,
+  ThornodeQuoteResponse,
+  ThornodeQuoteResponseSuccess,
+} from 'lib/swapper/swappers/ThorchainSwapper/types'
+import { THORCHAIN_FIXED_PRECISION } from 'lib/swapper/swappers/ThorchainSwapper/utils/constants'
+
+import { thorService } from '../thorService'
+
+export const getQuote = async ({
+  sellAsset,
+  buyAssetId,
+  sellAmountCryptoBaseUnit,
+  receiveAddress,
+  affiliateBps = '0',
+  deps,
+}: {
+  sellAsset: Asset
+  buyAssetId: AssetId
+  sellAmountCryptoBaseUnit: string
+  receiveAddress: string
+  affiliateBps?: string
+  deps: ThorchainSwapperDeps
+}): Promise<Result<ThornodeQuoteResponseSuccess, SwapErrorRight>> => {
+  const buyPoolId = adapters.assetIdToPoolAssetId({ assetId: buyAssetId })
+  const sellPoolId = adapters.assetIdToPoolAssetId({ assetId: sellAsset.assetId })
+
+  const sellAmountCryptoPrecision = bn(sellAmountCryptoBaseUnit).div(
+    bn(10).pow(sellAsset.precision),
```

Comment:
we can use `baseUnitToHuman` if you prefer

Reply:
Addressed in https://github.com/shapeshift/web/pull/4361/commits/14a26c4702a6da0f999cdb4a3987ad25bf7091b2.